shared/macros/10-bash.jinja

```diff
@@ -1834,3 +1834,147 @@ if ! $found ; then
     echo -e "[{{{ section }}}]\n{{{ key }}} = {{{ value }}}" >> "$file"
 fi
 {{%- endmacro %}}
+
+
+{{#
+    Make sure that a line with a specific PAM module is present with the correct control.
+    If the line is not present, it will be included after the regex informed in the "after_match"
+    parameter. If the "after_match" parameter is empty, the line will be included at the end of
+    the file informed in the "pam_file" parameter.
+    If the line was already present, but with a different control, the control will be updated.
+    Note: If there are multiple lines matching the "group" + "module", only the first line will be
+    updated. This is a conservative safeguard for improper use of this macro in rare cases of
+    modules configured by multiple lines, like pam_sss.so and pam_faillock.so. In some situations,
+    these special modules may have similar lines sharing the same "group" and "module". For these
+    specific cases, this macro is not recommened and a custom remediation should be considered.
+
+:param pam_file:        PAM config file.
+:param group:           PAM management group: auth, account, password or session.
```

Comment:
From PAM man:
```
Linux-PAM separates the tasks of authentication into four independent management groups: account management; authentication management; password management; and session management.
```
So, seem `group` is a better wording than `type`.

Reply:
From pam.conf:
```
       The format of each rule is a space separated collection of tokens, the first three being case-insensitive:

        service type control module-path module-arguments

...

       The type is the management group that the rule corresponds to. It is used to specify which of the management groups the subsequent module is to be  associated with.

account
           this module type performs non-authentication ...
 auth
           this module type ..
...

 If the type value from the list above is prepended with ...

```

It uses `module type` to refer to account, auth, password and session.

Reply:
So, both are correct and mean the same. While `management group` is the the more specific and also mentioned in both man pages (pam and pam.conf), I still prefer it. It is not wrong, but I can update the macro description to also mention `type` instead of remove `group`.

Reply:
I still keep my suggestion of changing it to `type`, but I`m fine as is. :)